lab_1_classify_by_unigrams/main.py

```diff
@@ -63,6 +132,21 @@ def detect_language(
     :param profile_2: a dictionary of a known profile
     :return: a language
     """
+    if not isinstance(unknown_profile, dict):
+        return None
+    if not isinstance(profile_1, dict) or not isinstance(profile_2, dict):
+        return None
+    mse_1 = compare_profiles(unknown_profile, profile_1)
+    mse_2 = compare_profiles(unknown_profile, profile_2)
+    if mse_1 == mse_2:
+        names = [profile_1['name'], profile_2['name']]
+        names.sort()
+        language = names[0]
```

Comment:
Consider directly returning here and in the next condition: this way code is more readable and there are fewer chances to make a mistake with variables.

Reply:
fixed